config/options.go

```diff
@@ -225,6 +227,11 @@ type Options struct {
 	// DataBrokerURL is the routable destination of the databroker service's gRPC endpiont.
 	DataBrokerURLString string   `mapstructure:"databroker_service_url" yaml:"databroker_service_url,omitempty"`
 	DataBrokerURL       *url.URL `yaml:",omitempty"`
+	// DataBrokerBackendStorageType is the storage backend type that databroker will use.
+	// Supported type: memory, redis
+	DataBrokerBackendStorageType string `mapstructure:"databroker_backend_storage_type" yaml:"databroker_backend_storage_type,omitempty"`
+	// DataBrokerBackendStorageConnectionString is the data source name for storage backend.
+	DataBrokerBackendStorageConnectionString string `mapstructure:"databroker_backend_storage_connection_string" yaml:"databroker_backend_storage_connection_string,omitempty"`
```

Comment:
@calebdoxsey will these fields need to be updated in the settings pb definitions?

Reply:
Yes if we change the settings in pomerium we also have to change them in the dashboard.

Reply:
Not sure it's necessary, I guess we only store policies and routes config in databroker?